images/leap/jeos/profiles.yaml

```diff
@@ -0,0 +1,5 @@
+profiles:
+  default:
+    description: Default Profile
+    include:
+      - csp
```

Comment:
That doesn't do anything. There should be a profile that includes `base/jeos` as bare minimum, and CSP specific ones if you want to create images for public clouds.